optaplanner-examples/src/main/resources/org/optaplanner/examples/meetingscheduling/optional/score/meetingSchedulingConstraints.drl

```diff
@@ -24,8 +24,19 @@ global HardMediumSoftScoreHolder scoreHolder;
 
 rule "Room conflict"
     when
-        $leftAssignment : MeetingAssignment(room != null, $leftId : id, $room : room)
-        $rightAssignment : MeetingAssignment(room == $room, calculateOverlap($leftAssignment) > 0, id > $leftId)
```

Comment:
Because the new implementation gives DRL the option of properly index the operations,as opposed to just getting some opaque predicate.
We keep the method, because it is useful in the consequence, where we actually just care about the number.